packages/icons/src/leg-icon.js

```diff
@@ -0,0 +1,77 @@
+import { legType } from "@opentripplanner/core-utils/lib/types";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Biketown from "./companies/biketown-icon";
+import Bird from "./companies/bird-icon";
+// import Bolt from "./companies/bolt-icon";
+// import Car2go from "./companies/car2go-icon";
+import Gruv from "./companies/gruv-icon";
+import Lime from "./companies/lime-icon";
+// import Lyft from "./companies/lyft-icon";
```

Comment:
Why are these icons commented out?

Reply:
I don't know. They were that way initially. We can add them back.